packages/manager-components/src/components/action-banner/action-banner.component.tsx

```diff
@@ -41,15 +44,28 @@ export function ActionBanner({
             }}
           ></span>
         </OsdsText>
-        <OsdsButton
-          className="sm:mt-0 mt-4 sm:ml-4 ml-0"
-          data-testid="actionBanner-button"
-          size={ODS_BUTTON_SIZE.sm}
-          color={ODS_THEME_COLOR_INTENT.primary}
-          onClick={onClick}
-        >
-          {cta}
-        </OsdsButton>
+        {onClick && (
+          <OsdsButton
+            className="sm:mt-0 mt-4 sm:ml-4 ml-0"
+            data-testid="actionBanner-button"
+            size={ODS_BUTTON_SIZE.sm}
+            color={ODS_THEME_COLOR_INTENT.primary}
+            onClick={onClick}
+          >
+            {cta}
+          </OsdsButton>
+        )}
+        {href && (
+          <OsdsLink
+            className="sm:mt-0 mt-4 sm:ml-4 ml-0"
+            color={ODS_THEME_COLOR_INTENT.primary}
+            onClick={() => onClick && onClick()}
+            href={href}
+            target={OdsHTMLAnchorElementTarget._blank}
+          >
+            {cta}
+          </OsdsLink>
+        )}
```

Comment:
So if there is an href and onClick there will be an OsdsButton and an OsdsLink ?

Reply:
Component updated, and issue resolved